specs/configure_monitoring.adoc

```diff
@@ -240,6 +240,28 @@ will look like:
 
 ----
 namespace.tendrl.node_monitoring:
+  objects:
+    Tendrl_definition:
+      attrs:
+        data:
+          help: "The flow definitions corresponding to performance_monitoring module"
+          type: json
+        value:
```

Comment:
looks good, I missed mentioning in last review, this is how the yaml should look

> namespace.tendrl.node_monitoring:
>   objects:
>     Tendrl_definition:
>       value:/_tendrl/definitions/integrations.d/performance_monitoring/data
>       attrs:
>         data:
>           help: "The flow definitions corresponding to performance_monitoring module"
>           type: json
>     Config:
>         value: _tendrl/config/performance_monitoring
>         attrs:
>         data:
>           help: "The flow definitions corresponding to performance_monitoring module"
>           type: json
> tendrl_schema_version: 0.3
>